README.md

```diff
@@ -89,7 +89,10 @@ limited by the capacity of a single physical machine, lest we sacrifice the
 property of complete auditability.  The Lightning Network [\[7\]][7] can help
 scale Bitcoin transaction volume by leaving some transactions off the ledger
 completely and is well suited for micropayments and privacy preserving payment
-rails, but may not be suitable for more generalized scaling needs.
+rails, and it is suitable for more generalized scaling needs.
+<!---
+state-channels allow for generalized smart contracts in the lightning network
+--->
```

Comment:
If you and I have a channel with $1000 in it, then we couldn't have more than $1000 of anything in our channel.
We could make it have $100 linked to the price of gold, and $100 that moves inverse to the price of gold, and they have margins.
We would have $800 left to make other types of contracts.